framework/include/outputs/CSV.h

```diff
@@ -97,6 +97,15 @@ class CSV : public TableOutput
 
   /// Flag indicating MOOSE is recovering via --recover command-line option
   bool _recovering;
+
+  /// Flag for creating a _FINAL symlink
+  bool _create_final_symlink;
+
+  /// Flag for creating a _LATEST symlink
+  bool _create_latest_symlink;
+
+  /// Current list of VPP filenames (see CSV::output for more information)
+  std::vector<std::pair<std::string, std::string>> _latest_vpp_filenames;
```

Comment:
Since this is a pair, can you be clear about what the pair contains? I gather the name and the link? maybe an example right here would be very helpful. It's unclear why the latest is a pair.